src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/RequiredReferenceInputArgumentCollection.java

```diff
@@ -11,7 +11,21 @@
 public final class RequiredReferenceInputArgumentCollection extends ReferenceInputArgumentCollection {
 
     @Argument(fullName = StandardArgumentDefinitions.REFERENCE_LONG_NAME, shortName = StandardArgumentDefinitions.REFERENCE_SHORT_NAME, doc = "Reference sequence file", optional = false)
-    private File referenceFile;
+    private String referenceFileName;
 
-    public File getReferenceFile(){ return referenceFile;}
+    private File referenceFile = null;
+
+    // synchronized to avoid a race in the check-then-modify sequence
+    @Override
+    public synchronized File getReferenceFile() {
```

Comment:
Here too, this synchronization seems unnecessary given the single-threadedness of hellbender.